Rocket.Chat/Controllers/Auth/AuthViewControllerAuthenticationHandler.swift

```diff
@@ -36,8 +38,6 @@ extension AuthViewController {
                 SocketManager.removeConnectionHandler(token: strongSelf.socketHandlerToken)
 
                 if let user = resource.user {
-                    BugTrackingCoordinator.identifyCrashReports(withUser: user)
```

Comment:
@cardoso @filipealva This method is called on `reloadApp` right after this call... do you guys think we should keep it?

Reply:
@rafaelks I've put it there as a redundancy, but since `reloadApp` is almost always called we don't need this call. Actually, after investigating here I saw that we may need to add this call on `didFinishLaunchingWithOptions`, because it is the only place that starts the chat without calling `reloadApp` (if it is already authenticated).